.travis.yml

```diff
@@ -27,6 +28,7 @@ env:
     - PIP_CACHE=$HOME/.cache/pip
     - VOLATILE_DIR=/tmp
     - DD_CASHER_DIR=/tmp/casher
+    - INTEGRATIONS_CORE_BRANCH=jaime/pdhfixes
```

Comment:
Once https://github.com/DataDog/integrations-core/pull/1183 is merged - switch this back to `master`